extra/make-cfg.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+#  This script will generate config for make-deb and deployment on bintray in TravisCI, 
```

Comment:
what kind of config ?
actually,  "make-deb" only use the exported DEBBRANCH variable, the config(.bintray.json) is used for TravisCI